clients/go/pkg/worker/jobDispatcher_test.go

```diff
@@ -188,13 +188,19 @@ func (suite *JobDispatcherSuite) completeJob(syncHandler bool) {
 
 type jobClientStub struct {
 	invoked bool
+	errorThrown bool
```

Comment:
:x: As the property isn't used for the tests, you can just omit this.

Reply:
done